app/helpers/freecite_helper.rb

```diff
@@ -0,0 +1,125 @@
+require 'uri'
+require 'nokogiri'
+
+# TODO TLNI: Move these classes (They are not Helpers ... But where to?)
```

Comment:
They could live under /lib/freecite. This is the last stop stuff goes before it's extracted into a gem.